src/main/java/com/google/sps/data/Event.java

```diff
@@ -72,15 +72,15 @@
   public Event(Entity entity) {
     this.id = (long) entity.getKey().getId();
     this.name = (String) entity.getProperty("eventName");
-    this.ownerOrgId = (long) entity.getProperty("eventOwnerOrgId");
+    //this.ownerOrgId = (long) entity.getProperty("eventOwnerOrgId");
```

Comment:
dont we want to keep this?

Reply:
I left this in here as a comment because I wasn't sure what to do about it- it was giving me an error with trying to cast to long here

Reply:
If you intend to submit like this (prior to the event class PR) please leave a comment that you are doing this on purpose and why.